vita3k/gui/src/home_screen.cpp

```diff
@@ -233,6 +233,8 @@ static bool app_filter(const std::string &app) {
     };
 
     switch (app_region) {
+    case ALL:
```

Comment:
you have forget add "default:" before

Reply:
There is no need for default because there is not a missing case there, all the cases are present so its not needed

Reply:
i see